src/controllers/controllerengine.cpp

```diff
@@ -835,13 +835,25 @@ void ControllerEngineConnectionScriptValue::disconnect() {
    Input:   the ControllerEngineConnection to trigger
    -------- ------------------------------------------------------ */
 void ControllerEngine::triggerControl(const ControllerEngineConnection conn) {
-    ControlObjectScript* coScript = getControlObjectScript(conn.key.group, conn.key.item);
+    if (m_pEngine == nullptr) {
+        return;
+    }
 
-    if (m_pEngine == nullptr || coScript == nullptr) {
+    ControlObjectScript* coScript = getControlObjectScript(conn.key.group, conn.key.item);
+    if (coScript == nullptr) {
         return;
     }
 
-    coScript->emitValueChanged();
+    QScriptValueList args;
```

Comment:
To reduce duplication -- want to make this a helper function on ControllerEngineConnection and call it from here and ControlObjectScript?